### ISSUE TITLE
Stop asking for a truffle config file when using aragonCLI inside an Aragon app repo

### ISSUE THREAD

**Issue by 0xGabi** [aragon/aragon-cli#1688]
## 🐛 Bug Report

If we use the commands on the buidler plugin we get a truffle.js don't find an error.  We shouldn't make this a hard requirement.

**Comment by macor161** [comment]
Indeed, this is an annoying bug for many users. It should be fixed as soon as we deprecate the app development features from aragonCLI
